packages/extension-ui/src/hooks/useGenesisHashOptions.ts

```diff
@@ -45,8 +52,9 @@ export default function (): Option[] {
           text: chain,
           value: genesisHash
         }))
```

Comment:
This time it would be better to use a reduce to create an array of 3 arrays - fist with aleph, second with relay and third with the rest. Having 5 operations on `chain` and spreading it 5 times seems like something we can avoid for performance gain.